ouroboros-consensus/test-consensus/Test/Consensus/Mempool.hs

```diff
@@ -471,18 +471,24 @@ prop_TxSeq_lookupByTicketNo_complete xs =
                    (zipWith TxTicket xs (map TicketNo [0..]))
 
 -- | Only finds elements in the sequence
-prop_TxSeq_lookupByTicketNo_sound :: [Positive Int] -> Positive Int -> Property
-prop_TxSeq_lookupByTicketNo_sound raw_xs (Positive k) =
+prop_TxSeq_lookupByTicketNo_sound ::
```

Comment:
Can you move these changes to the test to the other commit that adds the test? Then we'll have one commit adding the (failing test) and one commit that just fixes the bug.

Reply:
Done